critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -10,17 +10,21 @@
         {{ member.name }}
       </a>
       {% if member.periods %}
-        {% if not member.ended and member.periods and member.periods[0][0] %}
-          <span class="text-muted">
-            ({{ 'since ' + member.periods[0][0] }})
-          </span>
-        {% else %}
           <span class="text-muted">
           ({% for period in member.periods -%}
-            {{ period[0] + '-' + period[1] + ', ' * (not loop.last) }}
+              {% if not loop.last %}
```

Comment:
can you include a screenshot of each of these examples? (time period with _since_, time period with _until_, something with more than 1 period, and something with more than one period, and one of the since/until examples).
What would be great is if we could construct a `member.periods` list and render this section, checking the resulting html. I'm not sure if there is a method in jinja or flask-test to check the contents of an html template while ignoring additional whitespace. This could be interesting to look into.

Reply:
I have attached a screenshot. I couldn't get one for the until because I only have the sample dump. I'll try to get a full dump and try again but it'll take some time. Also, I could not understand the rest of your comment specifically this part
>  if we could construct a member.periods list and render this section, checking the resulting html.
Do you mean that this processing should be done in the python file instead of the template or something else ?
![Screenshot from 2020-09-17 02-26-42](https://user-images.githubusercontent.com/27751938/93391917-8b567e00-f88d-11ea-9bbe-57f3791bb91c.png)